src/libraries/Native/Unix/Common/pal_io_common.h

```diff
@@ -90,7 +90,7 @@ inline static int32_t Common_Poll(PollEvent* pollEvents, uint32_t eventCount, in
     }
     else
     {
-        pollfds = calloc(eventCount, sizeof(*pollfds));
+        pollfds = (struct pollfd*) calloc(eventCount, sizeof(*pollfds));
```

Comment:
nit: match formatting as the code above
```suggestion
        pollfds = (struct pollfd*)calloc(eventCount, sizeof(*pollfds));
```